tests/parser/ScheduleTests.cpp

```diff
@@ -1338,6 +1338,7 @@ DATES             -- 1
 )";
 
     const auto& schedule = make_schedule(input);
+    static_cast<void>(schedule); // Silence unused variable warning.
```

Comment:
Any reason to not just disable this entire check completely?  It doesn't actually do anything as far as I can tell.  I'd probably just put `#if 0` ahead of `BOOST_AUTO_TEST_CASE` and `#endif` after the tests's closing brace.